src/main/java/imsam/Command.java

```diff
@@ -1,31 +1,76 @@
 package imsam;
 
 import org.apache.logging.log4j.Level;
+import org.kohsuke.args4j.CmdLineParser;
 import org.kohsuke.args4j.Option;
+import org.kohsuke.args4j.OptionHandlerFilter;
 
+/**
+ * Implementations of this abstract class are used to create the command
+ * hierarchy of the program. Work is performed by the exec() method. The
+ * entryPoint() method is called when passing control to a next level
+ * subcommand. Generic argument flags, such as logging directives, are
+ * implemented here to allow them to be inserted anywhere in the argument
+ * hierarchy.
+ */
 public abstract class Command {
 
     @Option(name="-v",usage="enable verbose logging (INFO level)")
-    protected boolean verboseInfo = false;
+    protected boolean verboseLogging1 = false;
 
     @Option(name="-vv",usage="enable verbose logging (DEBUG level)")
-    protected boolean verboseDebug = false;
+    protected boolean verboseLogging2 = false;
 
     @Option(name="-vvv",usage="enable verbose logging (TRACE level)")
-    protected boolean verboseTrace = false;
+    protected boolean verboseLogging3 = false;
```

Comment:
I feel like using the variables verboseInfo, verboseDebug, and verboseTrace is easier to understand immediately than 1,2, and 3, but I'm fine with either set of variables